doc/ko/README.md

```diff
@@ -270,9 +355,11 @@ flame-graph 형식의 결과 또한 지원한다. 해당 데이터는 `uftrace d
 제약사항
 ===========
 - 리눅스에서 실행되는 네이티브 C/C++ 사용자 프로그램에 대해서만 사용 가능하다.
```

Comment:
This point need to be update `It can trace a C/C++/Rust/Python application on Linux and Android.`